### DISCUSSION TITLE
Moving fast without comprising

### DISCUSSION THREAD

**Discussion by gcavanunez** [tailwindlabs/tailwindcss#3203] [Kind words]
It’s been the first full year that I’ve embraced using utility first and man it’s been so useful.

I don’t have to compromise with the design team on ui elements due to time or juggle a bunch of files since out of the box during development all of them are right there to use. Don’t have to waste time with complex sass/stylu/(insert preprocessor) pipelines.

The adoption isn’t even strict to new projects, legacy apps can incrementally be updated with ease. The purge clean up is another one of those easy wins and can not just pass over the fact the team putting this tool together is also passionate about the product.

Throughout this year I’ve use it with legacy bootstrap, material and in-house  styled applications, the incremental adoption really aided me in reducing complexity and unnecessary classes that just bloated. 

Thank you so much for your work, it’s truly made my dev experience be much more enjoyable.

**Comment by adamwathan** [comment]
Thank you so much @gcavanunez!
